plugins/by-name/nui/default.nix

```diff
@@ -0,0 +1,11 @@
+{ lib, ... }:
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "nui";
+  originalName = "nui.nvim";
+  package = "nui-nvim";
+  url = "https://github.com/MunifTanjim/nui.nvim/";
```

Comment:
Most packages have a `meta.homepage`. If this one does (I haven't checked), we don't need to set `url` ourselves.